README.md

```diff
@@ -12,24 +12,47 @@
         --schema-handler-package "<LINK_TO_S3>"
     ```
 
-    With the following placeholders:
+2. View the version of the newly registered resource by running the following in your terminal:
+
+    ```shell
+    aws cloudformation list-type-versions \
+    --region "<REGION>" \
+    --type RESOURCE \
+    --type-name "<DATADOG_RESOURCE_NAME>"
+    ```
+
+3. Set this newly registered version as the `default` by running the following in your terminal:
+
+    ```shell
+    aws cloudformation set-type-default-version \
+        --region "<REGION>" \
+        --type RESOURCE \
+        --version-id <VERSION_ID> \
+        --type-name "<DATADOG_RESOURCE_NAME>" \
+    ```
+
+    With the following required placeholders:
     * `<REGION>`: Your AWS region.
-    * `<DATADOG_RESOURCE_NAME>`: The name of the resource to register, refer to the table below to see the supported resources.
+    * `<DATADOG_RESOURCE_NAME>`: The name of the resource to register, refer to the [table below](#resources-available) to see the Datadog supported resources.
     * `<LINK_TO_S3>`: S3 link to the resource.
-      * S3 link: `s3://datadog-cloudformation-resources/<RESOURCE_FOLDER>/<RESOURCE_FOLDER>-<VERSION>.zip`
+      * S3 link: `s3://datadog-cloudformation-resources/<RESOURCE_NAME>/<RESOURCE_NAME>-<RESOURCE_VERSION>.zip`
```

Comment:
Actually, it is the resource folder, as in what's in the folder column in the table below
```suggestion
      * S3 link: `s3://datadog-cloudformation-resources/<RESOURCE_FOLDER>/<RESOURCE_FOLDER>-<RESOURCE_VERSION>.zip`
```

Reply:
I had missed that it was called that below 🤦‍♂